intelmq/bots/parsers/fireeye/parser.py

```diff
@@ -0,0 +1,94 @@
+
+"""
+Fireeye Parser Bot
+Retrieves a base64 encoded JSON-String from raw and converts it into an
+event.
+""" 
+from intelmq.lib import utils
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.message import MessageFactory
+from intelmq.lib.utils import base64_decode
+import json
+import xmltodict
+
+
+import ipaddress
+
+class FireeyeParserBot(ParserBot):
+
+    def process(self):
+        report = self.receive_message()
+        raw_report = utils.base64_decode(report.get('raw'))
+        my_dict = xmltodict.parse(raw_report)
+        try:
+            event = self.new_event(report)
+            for indicator in my_dict['OpenIOC']['criteria']['Indicator']['IndicatorItem']:
+                hashValue = indicator['Content']['#text']
+                indicatorType = indicator['Context']['@search']
+                if indicatorType == 'FileItem/Md5sum':
+                   event.add('malware.hash.md5', indicator['Content']['#text'])
+                if indicatorType == 'FileItem/Sha256sum':
+                   self.logger.debug('#######FileItem/Md5sum aus uuid########'+ indicator['Content']['#text'])
```

Comment:
Even if it is DEBUG log level, please don't use multiple hashes for styling.

Reply:
removed